bounded_counter/tests/count_min_sketch_test.py

```diff
@@ -0,0 +1,25 @@
+import bounded_counter
+from timeit import default_timer as timer
+import cProfile
```

Comment:
Output of a  line profiler (~kernprof) might be more illuminating for profiling hotspots.